akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -227,9 +227,6 @@ private[remote] class Remoting(_system: ExtendedActorSystem, _provider: RemoteAc
       s"Attempted to quarantine address [$remoteAddress] with uid [$uid] but Remoting is not running", null)
   }
 
-  // Not used anywhere only to keep compatibility with RemoteTransport interface
-  protected def useUntrustedMode: Boolean = provider.remoteSettings.UntrustedMode
-
```

Comment:
let's say we want to merge this into master later and release something together with Akka 2.4.x, should we try to not touch things that would break then?